app/views/shared/_mailer_header.html.erb

```diff
@@ -0,0 +1,7 @@
+<div id="header-container" class="grid">
+  <div id="header-content" class="grid">
+    <%= image_tag(attachments['logo@2x.png'].url) %>
+    <h2 id="header-title">a community for mental health experiences</h2>
```

Comment:
We should be using i18n here, let's use `<h2 id="header-title">- <%= t('app_description') %></h2>`.